lib/dpl/provider/pypi.rb

```diff
@@ -64,7 +65,9 @@ def needs_key?
 
       def push_app
         context.shell "python setup.py register -r #{options[:server] || 'pypi'}"
```

Comment:
Yea, quite possibly. Not sure I ever read the docs but currently the only safe way is to do a one time register via the website (can just use version 0 and delete it) and then just use ``twine upload`` in the future.